.circleci/config.yml

```diff
@@ -0,0 +1,56 @@
+version: 2
+
+defaults: &defaults
+  docker:
+    - image: circleci/python:3.6
+  working_directory: ~/marquez_airflow
+
+
+jobs:
+  build:
+    <<: *defaults
+    steps:
+      - checkout
+      - run: pip install --user  -r requirements.txt
+
+  test:
+    <<: *defaults
+    steps:
+      - checkout
+      - run: pip install --user  -r test-requirements.txt
+      - run:
+          name: Install airflow from package
+          environment:
+            SLUGIFY_USES_TEXT_UNIDECODE: 'yes'
+          command: pip install --user -e .
+      - run: python -m pytest ./test
+
+  integration-test:
+    <<: *defaults
+    steps:
+      - checkout
+      - run: pip install --user -r test-requirements.txt
+
+  release:
+    <<: *defaults
+    working_directory: ~/marquez_airflow
```

Comment:
Do we need the working_directory here again, if it was defined in the defaults already?